### PR TITLE
changing time to unix time

### PR DESCRIPTION
**Pull request by fvseto**

### CODE REVIEW

File: backendprocesses.cpp [hunk 1]
```diff
@@ -84,14 +84,23 @@ void BackendProcesses::threadProcedure()
...
     bytes.insert(tstampOffsets.hr, hour_time & 0xFF);
-    bytes.remove(tstampOffsets.mn,1);
-    bytes.insert(tstampOffsets.mn, min_time & 0xFF);
-    bytes.remove(tstampOffsets.sc,1);
-    bytes.insert(tstampOffsets.sc, sec_time & 0xFF);
-    bytes.remove(tstampOffsets.ms,2);
-    bytes.insert(tstampOffsets.ms, msec_time & 0xFF);
-    bytes.insert(tstampOffsets.ms, (msec_time >> 8) & 0xFF);
-
+  //  bytes.remove(tstampOffsets.mn,1);
```
Comment (MCLiii) [inline backendprocesses.cpp]: Need to uncomment those lines

File: backendprocesses.cpp [hunk 2]
```diff
@@ -84,14 +84,23 @@ void BackendProcesses::threadProcedure()
...
+   // bytes.insert(tstampOffsets.sc, sec_time & 0xFF);
+  //  bytes.remove(tstampOffsets.ms,2);
+ //   bytes.insert(tstampOffsets.ms, msec_time & 0xFF);
+  //  bytes.insert(tstampOffsets.ms, (msec_time >> 8) & 0xFF);
+   // qDebug()<< (hour_time&0xFF);
+   // qDebug()<< (curr_msec/10);
+    uint64_t time=curr_msec;
+    int mask= 56;
+    for(int i=0; i<=7; i++) {
+        int print= curr_msec;
```
Comment (MCLiii) [inline backendprocesses.cpp]: Remove this line

File: backendprocesses.cpp [hunk 3]
```diff
@@ -84,14 +84,23 @@ void BackendProcesses::threadProcedure()
...
+        qDebug()<<"t="<< ((time>>mask) & 0xFF);
+        bytes.insert(tstampOffsets.ms,(time>>mask) & 0xFF);
```
Comment (MCLiii) [inline backendprocesses.cpp]: Instead of tstampOffsets.ms would need to add a Instead of tstampOffsets.unix to the struct and use that as the unix offset

**Review by MCLiii — COMMENTED** [review]
Should be ready to merge after you addressed those changes

File: backendprocesses.cpp [hunk 4]
```diff
@@ -84,14 +84,23 @@ void BackendProcesses::threadProcedure()
...
+  //  bytes.remove(tstampOffsets.ms,2);
+ //   bytes.insert(tstampOffsets.ms, msec_time & 0xFF);
+  //  bytes.insert(tstampOffsets.ms, (msec_time >> 8) & 0xFF);
+   // qDebug()<< (hour_time&0xFF);
+   // qDebug()<< (curr_msec/10);
+    uint64_t time=curr_msec;
+    int mask= 56;
+    for(int i=0; i<=7; i++) {
+        int print= curr_msec;
+        qDebug()<<"t="<< ((time>>mask) & 0xFF);
```
Comment (MCLiii) [inline backendprocesses.cpp]:
```suggestion

```
Remove unnecessary outputs

File: backendprocesses.cpp [hunk 5]
```diff
@@ -84,14 +84,23 @@ void BackendProcesses::threadProcedure()
...
-
+  //  bytes.remove(tstampOffsets.mn,1);
+  //  bytes.insert(tstampOffsets.mn, min_time & 0xFF);
+   // bytes.remove(tstampOffsets.sc,1);
+   // bytes.insert(tstampOffsets.sc, sec_time & 0xFF);
+  //  bytes.remove(tstampOffsets.ms,2);
+ //   bytes.insert(tstampOffsets.ms, msec_time & 0xFF);
+  //  bytes.insert(tstampOffsets.ms, (msec_time >> 8) & 0xFF);
+   // qDebug()<< (hour_time&0xFF);
+   // qDebug()<< (curr_msec/10);
```
Comment (jrvollmer) [inline backendprocesses.cpp]: Commented out debug statements can be removed

File: backendprocesses.h [hunk 6]
```diff
@@ -16,6 +16,7 @@ struct timestampOffsets {
     int mn;
     int sc;
     int ms;
+    int unix;
```
Comment (MCLiii) [inline backendprocesses.h:19]: You will need to go to dataunpacker.cpp and set the offset for this variable

**Review by MCLiii — APPROVED** [review]
Approved

**Pull request merged**